git-setup.sh

```diff
@@ -23,6 +23,10 @@ for module in "$@"; do
     git submodule add --force $project "apps/$module"
     # Set the default branch to main
     git config -f .gitmodules --add "submodule.apps/$module.branch" main
+
+    # Update to the latest from main in that submodule
+    cd apps/$module && git pull origin main && cd ../..
```

Comment:
What would happen if you're on a different git branch for a given submodule? I think it will simply error, right? Which might be okay. Mainly hoping it wouldn't discard any open changes here.